packages/core/src/state/NetworkManager.ts

```diff
@@ -136,7 +136,8 @@ export default class NetworkManager implements Manager {
 
   protected getLastReset() {
     const lastReset = this.getState().lastReset;
-    if (typeof lastReset !== 'number') return lastReset.getTime();
+    if (lastReset instanceof Date) return lastReset.valueOf();
```

Comment:
heads up that this can still return NaN, which isn't serializable.

Reply:
The return value of this isn't serialized. It's the state that must be serialized